workshops/models.py

```diff
@@ -227,6 +227,16 @@ def unpublished_events(self):
 
         return self.filter(published=False)
 
+    def unpaid_events(self):
+        '''Return a queryset for events that owe money.'''
+
+        # All events that have a non-zero admin fee.
+        queryset = self.filter(admin_fee__ne=0)
+
+        # Of those, all those whose paid status is not 'Yes' (i.e., is 'No' or NULL).
+        queryset = queryset.filter(fee_paid__ne=True)
```

Comment:
@wking you're totally right. I doubt Amy would be used for accounting, so no need to track payments, unless @gvwilson has some plans for that.